[2024-04-25T06:23:28.939+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-25T06:23:28.950+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_minio_s3_v02.sensor_minio_s3 scheduled__2023-12-18T00:00:00+00:00 [queued]>
[2024-04-25T06:23:28.957+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_minio_s3_v02.sensor_minio_s3 scheduled__2023-12-18T00:00:00+00:00 [queued]>
[2024-04-25T06:23:28.958+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-25T06:23:28.969+0000] {taskinstance.py:2327} INFO - Executing <Task(S3KeySensor): sensor_minio_s3> on 2023-12-18 00:00:00+00:00
[2024-04-25T06:23:28.974+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=13150) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-25T06:23:28.976+0000] {standard_task_runner.py:63} INFO - Started process 13155 to run task
[2024-04-25T06:23:28.974+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_minio_s3_v02', 'sensor_minio_s3', 'scheduled__2023-12-18T00:00:00+00:00', '--job-id', '2553', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_minio_s3.py', '--cfg-path', '/tmp/tmpgdf1ip0a']
[2024-04-25T06:23:28.977+0000] {standard_task_runner.py:91} INFO - Job 2553: Subtask sensor_minio_s3
[2024-04-25T06:23:29.012+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_minio_s3_v02.sensor_minio_s3 scheduled__2023-12-18T00:00:00+00:00 [running]> on host a55babecb56c
[2024-04-25T06:23:29.075+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_minio_s3_v02' AIRFLOW_CTX_TASK_ID='sensor_minio_s3' AIRFLOW_CTX_EXECUTION_DATE='2023-12-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-18T00:00:00+00:00'
[2024-04-25T06:23:29.076+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-25T06:23:29.084+0000] {baseoperator.py:399} WARNING - S3KeySensor.execute cannot be called outside TaskInstance!
[2024-04-25T06:23:29.085+0000] {s3.py:110} INFO - Poking for key : s3://***/data.csv
[2024-04-25T06:23:29.096+0000] {base.py:84} INFO - Using connection ID 'minio_conn' for task execution.
[2024-04-25T06:23:29.098+0000] {connection_wrapper.py:387} INFO - AWS Connection (conn_id='minio_conn', conn_type='aws') credentials retrieved from extra.
[2024-04-25T06:23:29.102+0000] {logging_mixin.py:188} WARNING - <string>:9 AirflowProviderDeprecationWarning: extra['host'] is deprecated and will be removed in a future release. Please set extra['endpoint_url'] instead
[2024-04-25T06:23:29.253+0000] {base.py:294} INFO - Success criteria met. Exiting.
[2024-04-25T06:23:29.256+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-25T06:23:29.271+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_with_minio_s3_v02, task_id=sensor_minio_s3, execution_date=20231218T000000, start_date=20240425T062328, end_date=20240425T062329
[2024-04-25T06:23:29.326+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-25T06:23:29.347+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-25T06:23:29.363+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
